content/posts/developing-versionless-graphql-apis.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Developing versionless APIs with GraphQL"
+date: "2020-07-22"
+author: "Bing Xiao"
+github: xo426017
+categories: ["graphql", "API"]
+featuredImage: "/images/graphql.png"
```

Comment:
To be consistent with the other posts, the graphql logo should be:

`/images/developing-versionless-graphql-apis/featured.png`